ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -111,9 +111,11 @@
     <li
```

Comment:
@cloudhead I could wrap the tooltip around the `<li>` and change the `ul` and `li` to divs so the markup is correct. But it's complicated cause some of the markup is replicated in the `ConnectionStatusIndicator` component.